redux-router/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "@types/react-router": "^2.0.41",
```

Comment:
It's available now. Here's what to do:
* Set `history/v2/tsconfig.json` to:
```json
{
    "compilerOptions": {
        "module": "commonjs",
        "target": "es6",
        "noImplicitAny": true,
        "noImplicitThis": true,
        "strictNullChecks": false,
        "baseUrl": "../../",
        "typeRoots": [
            "../../"
        ],
        "types": [],
        "paths": {
            "history": ["history/v2"],
            "history/*": ["history/v2/*"]
        },
        "noEmit": true,
        "forceConsistentCasingInFileNames": true
    },
    "files": [
        "index.d.ts",
        "history-tests.ts"
    ]
}
```

* In `react-breadcrumbs`, `react-i18next`, `react-router-bootstrap`, `react-router-redux`, `react-router-redux/v3`, `redux-router`, and `react-router`:
    - Undo all your changes (Don't add `"@types/history"` to `package.json`, undo `tsconfig.json` changes).
    - Add this to `tsconfig.json`:
      ```json
      "paths": {
          "history": ["history/v2"]
      },
      ```